pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -293,6 +293,14 @@
         "4.0.1.0": "4.6.0"
       }
     },
+    "System.Windows.Extensions": {
+      "InboxOn": {
+        "netcoreapp3.0": "4.6.4.0"
```

Comment:
I don't think it should be inbox, at least not in netcoreapp.  It'll likely be part of a different shared framework higher up which isn't tracked with this index.